example/package.json

```diff
@@ -10,7 +10,7 @@
     "react-dom": "^16.14.0",
     "react-live": "^2.2.3",
     "react-scripts": "^4.0.0",
-    "react-showtime": "https://github.com/azavea/react-showtime#develop",
+    "react-showtime": "https://github.com/azavea/react-showtime#feature/cek/release-workflow",
```

Comment:
This will be changed back to `develop` before merging, and changed to `0.1.0` as part of the subsequent release.